mysql-test/suite/rocksdb_stress/r/rocksdb_stress.result

```diff
@@ -1,3 +1,4 @@
+SET @@session.default_storage_engine = ROCKSDB;
```

Comment:
how is this related to the python change?

Reply:
This is required to run this test. It was failing before.